source/influxdb/api.d

```diff
@@ -799,6 +840,16 @@ struct InfluxValue {
         .to!string.shouldEqualLine(`cpu foo="bar" 7000000000`);
 }
 
+@("Measurement.to!string InfluxValue float string")
```

Comment:
There should probably be a few unittests constructing InfluxValues using the new constructors.